thirdeye-persistence/src/main/java/ai/startree/thirdeye/datalayer/mapper/MergedAnomalyIndexMapper.java

```diff
@@ -26,5 +29,19 @@ public interface MergedAnomalyIndexMapper {
 
   @Mapping(source = "enumerationItem.id", target = "enumerationItemId")
   @Mapping(target = "dimensions", ignore = true)
+  @Mapping(source = "anomalyLabels", target = "ignored", qualifiedByName = "labelsToIgnoredMapper")
   MergedAnomalyResultIndex toIndexEntity(AnomalyDTO dto);
+
+  @Named("labelsToIgnoredMapper")
+  static boolean labelsToIgnored(List<AnomalyLabelDTO> labels) {
```

Comment:
tip: Use IntelliJ to simplify. this is a single liner
```
return labels != null && labels.stream().anyMatch(AnomalyLabelDTO::isIgnore);
```

Reply:
That's beautiful! Made the changes.